tests/data/sync/pir_to_xy/100055064_12500_4800_10800.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Just a small suggestion, should we put somewhere the convention of the naming of those files ? `dataset_id_p_i_r` ?

Reply:
I included the script that generates these files in the description. IMO it is a good enough documentation. The actual name of the file is never used. It is the contents that matter.

Reply:
Sounds good for me!